### DISCUSSION TITLE
Thresholds Question

### DISCUSSION THREAD

**Discussion by ajhintz** [alexjustesen/speedtest-tracker#504] [Q&A] [answered]
I am trying to use the Thresholds notifications, but they don't seem to be working as I expect them to. I have 2gig service, so my speeds generally run between 1.9-2.3 Gbps. I want to setup a threshold notification so if my speeds drop below 1 Gbps, I get a notification. I have set it up that way (I think), but I am getting notified even when my speeds are above the notification threshold. Am I missing something?

Thanks!
<img width="639" alt="Screenshot 2023-03-20 at 2 45 05 PM" src="https://user-images.githubusercontent.com/54547836/226472960-d82311e9-3ab1-4509-9875-516820ebfdd8.png">

**Comment by alexjustesen** [comment, marked as answer]
Which version of Speedtest Tracker are you using? This should of been fixed in `v0.11.9`

**Reply by ajhintz** [reply]
That was it - I hadn't updated in a little while. Sorry I didn't try that first. Thanks for the response!
